src/user-guide/interventions/index.rst

```diff
@@ -439,3 +439,52 @@ Terminate a Chain of Automatic Retries
 Use the same intervention as :ref:`interventions.set-task-outputs`,
 but specify the "failed" output rather than
 "succeeded".
+
+
+Hold The Workflow And Trigger Tasks One By One
+----------------------------------------------
+
+:Example:
+   I want to hold back the workflow whilst I manually run one or more tasks
+   to fix a problem or test a task.
+
+:Solution:
+   * Hold all tasks after a specified cycle.
+   * Trigger the task(s) you want to run.
+   * When you're done triggering, release the held tasks to allow the workflow
+     to continue.
+
+.. tab-set::
+
+   .. tab-item:: GUI
+      :sync: gui
+
+      .. image:: set-and-release-hold-point.gif
```

Comment:
This is quite a lengthy GIF which could be halfway through playback when the user starts trying to follow along with it. Is it possible to embed a video instead?

Reply:
Dunno.

Reply:
Run the GIF through once, then stop and provide a button to play from the start is another option.